test/evaluation/inputDescriptorFilterEvaluationHandler.spec.ts

```diff
@@ -127,13 +167,20 @@ describe('inputDescriptorFilterEvaluationHandler tests', () => {
     message2.payload = { 'result': { 'path': ['$', 'issuer'], 'value': 'did:foo:123' }, 'valid': true };
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler: EvaluationHandler = new InputDescriptorFilterEvaluationHandler(evaluationClient);
-    evaluationHandler.handle(presentationDefinition, new VP(presentation));
+    evaluationHandler.handle(presentationDefinition, presentation);
     expect(evaluationClient.results).toEqual([message, message1, message2]);
   });
 
   it(`input descriptor's constraint.fields.filter match and nested_path`, () => {
     const inputCandidates: unknown = getFile('./test/dif_pe_examples/vp/vp_nested_submission.json');
-    const presentation: Presentation = new Presentation(inputCandidates['@context'], inputCandidates['presentation_submission'], inputCandidates['type'], inputCandidates['verifiableCredential'], inputCandidates['holder'], inputCandidates['proof']);
+    const presentation: VerifiablePresentation = {
+      '@context': inputCandidates['@context'],
+      presentationSubmission: inputCandidates['presentation_submission'],
+      type: inputCandidates['type'],
+      verifiableCredential: inputCandidates['verifiableCredential'],
+      holder: inputCandidates['holder'],
+      proof: inputCandidates['proof']
+    };
```

Comment:
instantiation should be cleaner with new

Reply:
Nope it is not. You are looking at this with a Java hat on